foods.html

```diff
@@ -5,7 +5,16 @@
   <title>Quantified Self - Manage Foods</title>
 </head>
 <body>
+  <h1>Manage Foods</h1>
 
+  <table id="foods-table">
+    <tr id="foods-table-header">
+      <th>Name</th>
+      <th>Calories</th>
+      <th></th>
+    </tr>
+  </table>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
I'm not sure if you prefer to use something else, but I defaulted to using Google's jQuery library.  Let me know if I should change it.

Reply:
I typically use a different jquery script, but I don't mind making use of the google's.  I can't foresee any issues, and if it turns out to be a problem - we'll cross that bridge when we come to it.

Reply:
So you shouldn't need to add this jQuery script as jquery is already included as a package in the webpack! Make use of it :)